plugins/codemirror/source.html

```diff
@@ -169,10 +177,11 @@
 	"use strict";
 
 	var idx = haystack.indexOf(needle), depth = 0, x;
-	for (x = idx; x >= 0; x--) {
+	for (x = idx -1; x >= 0; x--) {
```

Comment:
this is about setting cursor in codemirror inside &nbsp; or &uuml; or any similar entity.
w/o that code and you set cursor inside eg. &nbsp; and press ok in codemirror window, &nbsp; will be visible in tinymce...